### PR TITLE
Fix responsiveness

### PR DESCRIPTION
**Pull request by Nelson-Chinedu**

### CODE REVIEW

File: components/Dashboard/index.tsx [hunk 1]
```diff
@@ -38,12 +39,12 @@ const Dashboard: FunctionComponent<{}> = () => {
...
               type="button"
             >
-              Search
+              <Search className="w1"/>
             </Button>
           </div>
         </div>
       </section>
-      <section className="pv4 flex justfy-center">
+      <section className="c-ReccommendedMentorshipCardWrapper pv4 flex-m justify-between-m flex-l justify-between-l">
```
Comment (Hector101) [inline components/Dashboard/index.tsx]: `c-ReccommendedMentorshipCardWrapper ` can be renamed to just `c-mentorship-section`

File: components/Dashboard/index.tsx [hunk 2]
```diff
@@ -52,7 +53,7 @@ const Dashboard: FunctionComponent<{}> = () => {
         <div>
           <h3 className="mb0">Featured Mentors</h3>
         </div>
-        <div className="pv4 flex justfy-center">
+        <div className="c-FeaturedMentorCardWrapper pv4 flex-m justify-between-m flex-l justify-between-l">
```
Comment (Hector101) [inline components/Dashboard/index.tsx]: `c-FeaturedMentorCardWrapper` can be renamed to `c-featured-mentors-section`

File: components/Dashboard/index.tsx (hunk 1 above)
Comment (Nelson-Chinedu) [reply components/Dashboard/index.tsx]: Okay will work on the rename

Comment (Nelson-Chinedu) [reply components/Dashboard/index.tsx]: I just remove the custom class name. I noticed I didn't make use of it any longer

**Pull request merged**